lib/commands/exam/iterate.js

```diff
@@ -71,18 +80,21 @@ module.exports = {
    */
   _buildForTests() {
     // TODO: execa
-    var execSync = require('child_process').execSync;
+    const execSync = require('child_process').execSync;
```

Comment:
can we use `execa` here to avoid leaking child processes, as `execSync` does not cleanup child-processes always if the parent crashes or is killed.